dspace-api/src/main/java/org/dspace/app/launcher/ScriptLauncher.java

```diff
@@ -145,8 +146,13 @@ public static int handleScript(String[] args, Document commandConfigs,
     private static int executeScript(String[] args, DSpaceRunnableHandler dSpaceRunnableHandler,
                                      DSpaceRunnable script) {
         try {
-            script.initialize(args, dSpaceRunnableHandler, null);
-            script.run();
+            StepResult result = script.initialize(args, dSpaceRunnableHandler, null);
+            if (StepResult.Continue.equals(result)) {
+                // only run the script, if the normal initialize is successful
+                script.run();
+            } else {
```

Comment:
is this else really necessary?

Reply:
The else was only for documentation...

Else is removed the documentation refactored in 064e2caa37dfa283c3c08dee0e7321e36073bfa2